base/inc/fields/tinymce.class.php

```diff
@@ -517,14 +517,18 @@ private function render_media_buttons( $editor_id ) {
 		echo '<div id="wp-' . esc_attr( $editor_id ) . '-media-buttons" class="wp-media-buttons">';
 		
 		$screen = function_exists( 'get_current_screen' ) ? get_current_screen() : null;
-		// Temporarily disable the Jetpack Grunion contact form editor on the widgets screen.
-		if( ! is_null( $screen ) && $screen->id == 'widgets' ) {
-			remove_action( 'media_buttons', 'grunion_media_button', 999 );
-		}
-		do_action( 'media_buttons', $editor_id );
-		// Temporarily disable the Jetpack Grunion contact form editor on the widgets screen.
-		if( ! is_null( $screen ) && $screen->id == 'widgets' ) {
-			add_action( 'media_buttons', 'grunion_media_button', 999 );
+		if ( function_exists( 'grunion_media_button' ) && ! is_null( $screen ) ) {
```

Comment:
Can keep this as only 2 `if` statements and make it all a bit cleaner by assigning the boolean expression to a variable i.e.
```
        // Temporarily disable the Jetpack Grunion contact form editor on the widgets screen.
		$disable_grunion = function_exists( 'grunion_media_button' ) && ! is_null( $screen ) && $screen->id == 'widgets';
		if( $disable_grunion ) {
			remove_action( 'media_buttons', 'grunion_media_button', 999 );
		}
		do_action( 'media_buttons', $editor_id );
		// Temporarily disable the Jetpack Grunion contact form editor on the widgets screen.
		if( $disable_grunion ) {
			add_action( 'media_buttons', 'grunion_media_button', 999 );
		}
```